docker/config/test.env

```diff
@@ -30,17 +30,4 @@ OIDC_RP_CLIENT_SECRET=abcdef
 # This makes sure this is never a real valid URL.
 OIDC_OP_USER_ENDPOINT=https://auth.example.com/authorize
 
-SYMBOL_FILE_PREFIX=v0
```

Comment:
The symbol file prefix matches a file structure. It's wrong for the codebase to be using v1, but the tests are using v0. I removed this.